### PR TITLE
pretty formatting of the image listing

### PR DESCRIPTION
**Pull request by alansaviolobo**
updated the way photos are displayed.
linked them to the original photos listing page.

screenshot

### CODE REVIEW

File: fbphoto.module [hunk 1]
```diff
@@ -110,28 +109,62 @@ function theme_fbphoto_photo($vars) {
...
-      $attributes['src']    = $img->source;
-      $attributes['height'] = $img->height;
-      $attributes['width']  = $img->width;
-      $attributes['title']  = isset($photo->name) ? $photo->name : '';
-      $output = '<img '. drupal_attributes($attributes) .' />';
-      $output = '<a href="'. $photo->images[0]->source .'">'. $output .'</a>';
-      $items[] = '<span>'. $output .'</span>';
+      $bg_image = fbphoto_get_best_photo($photo);
+      $items[] =
+          l("<div style='background-image: url($bg_image);'></div>",
```
Comment (and1truong) [inline fbphoto.module:119]: Wow, very hacky :D

File: fbphoto.info [hunk 2]
```diff
@@ -3,4 +3,5 @@ version = "7.x-1.x-dev"
 core = 7.x
 description = "Filter for Facebook photo."
 
+files[] = fbphoto.inc
```
Comment (and1truong) [inline fbphoto.info:6]: Great, thanks :+1:

File: fbphoto.module (hunk 1 above)
Comment (alansaviolobo) [inline fbphoto.module:119]: we could run this through a theme function i guess ?

Comment (and1truong) [inline fbphoto.module:119]: I have added you add contributor, both github project and Drupal project, please fell free make changes. Thank you.

I think:
1. a tag should never wrap div tag
2. Inline CSS is… not nice :D

Comment (alansaviolobo) [inline fbphoto.module:119]: Thank you for adding me as a contributor. I will go ahead and merge the
pull request.

Regards,

**Pull request merged**